storage-proto/proto/transaction_by_addr.proto

```diff
@@ -121,7 +121,7 @@ enum InstructionErrorType {
     UNSUPPORTED_SYSVAR = 48;
     ILLEGAL_OWNER = 49;
     MAX_ACCOUNTS_DATA_SIZE_EXCEEDED = 50;
-    ACTIVE_VOTE_ACCOUNT_CLOSE = 51;
+    MAX_ACCOUNTS_EXCEEDED = 51;
```

Comment:
look likes scary merge conflict resolution. :)

Reply:
Haha, yeah we can just abandon that error code completely if this is too scary. It's unused now so thought I'd repurpose it

Reply:
I think repurposing things is fine, as long as it won't be back-ported.
But this PR is, so I agree with @ryoqun here.

Reply:
What does backporting have to do with this discussion?

Reply:
Backporting makes merge conflicts and mapping collisions more likely.

Reply:
That's true, in this case I don't expect it to be an issue. So unless you either of you feel strongly I'd like to move forward with repurposing this unused error code

Reply:
I've removed the backport to de-risk this change.